### PR TITLE
Wild function added

### PR DESCRIPTION
**Pull request by aditibhardwaj98**
Unit testing is done!
I request to please do it once more

### CODE REVIEW

File: src/cards_management.c [hunk 1]
```diff
@@ -66,16 +68,24 @@ int initialize_cards(void)
     }
     g_draw_pile->next = NULL;
 
-    for (i = RED; i <= YELLOW; i++)
+	int count_wild_card = 0;
+    for (i = RED; i <= BLACK; i++)
     {
+    	if(i == BLACK && count_wild_card != 2)
```
Comment (MiaYWei) [inline src/cards_management.c]: Please following the Coding conventions in the project requirement to cleanup all of your codd.

File: src/cards_management.c [hunk 2]
```diff
@@ -66,16 +68,24 @@ int initialize_cards(void)
     }
     g_draw_pile->next = NULL;
 
-    for (i = RED; i <= YELLOW; i++)
+	int count_wild_card = 0;
+    for (i = RED; i <= BLACK; i++)
```
Comment (MiaYWei) [inline src/cards_management.c:72]: Please add action card process in another function. Don't make modification which affects the normal card.

File: src/cards_management.c [hunk 3]
```diff
@@ -279,10 +289,22 @@ int get_pile_length(Deck_t *p_pile)
  */
 bool is_playable_card(Card_t card)
 {
-    if ((card.color == g_card_on_table.color) || (card.name == g_card_on_table.name))
-    {
-        return true;
-    }
+	if(is_wild_card_played)
```
Comment (MiaYWei) [inline src/cards_management.c:292]: Action card is always playable.

File: src/game.c [hunk 4]
```diff
@@ -25,7 +25,7 @@ void player_name_inquiry(void);
 void start_new_game(void)
 {
     player_name_inquiry();
-
+    bool is_confirmed = false;
```
Comment (MiaYWei) [inline src/game.c]: Unused code, already deleted. Why add it back? Please update your branch with the latest code from development branch.

File: src/human_player_module.c [hunk 5]
```diff
@@ -181,22 +182,32 @@ Card_t map_user_input(char *input)
...
     }
-
+	else if (input[0] == 'W')
+    {
+		
+		temp_card.name = WILD;
+		wild_card(HUMAN);
+		temp_card.color = g_card_on_table.color;
+    }
+
```
Comment (MiaYWei) [inline src/human_player_module.c]: Please cleanup your code by following coding conventions

File: src/game.c (hunk 4 above)
Comment (aditibhardwaj98) [reply src/game.c]: Line 28 has been removed.

File: src/cards_management.c (hunk 2 above)
Comment (aditibhardwaj98) [reply src/cards_management.c:72]: The action cards need to be initialized in the initialize_cards() function. The action card is defined seperately.

File: src/human_player_module.h [hunk 6]
```diff
@@ -13,6 +13,7 @@ void show_cards_assigned(Card_t assigned_card);
 bool check_is_valid_turn(Card_t *current_card);
 bool is_human_card(Card_t* current_card);
 void display_player_turn(PlayerType_e next_player);
+void wild_card(PlayerType_e type);
```
Comment (aditibhardwaj98) [inline src/human_player_module.h:16]: the warning which appeared has been taken care of here

**Comment by MiaYWei** [comment]
Inappropriate implementation, so we discard this pull request.

**Pull request closed without merging**